app.py

```diff
@@ -36,12 +36,13 @@
 def create_app():
     app = Quart(__name__)
     app.register_blueprint(bp)
+    app.config["TEMPLATES_AUTO_RELOAD"] = True
     return app
 
 
 @bp.route("/")
 async def index():
-    return await render_template("index.html", title=UI_TITLE, favicon=UI_FAVICON)
+    return await render_template("index.html")
```

Comment:
Let's not modify this line - we need the title and favicon args here for the UI customization.